test/mason/publish-tests/publishGitCheck.chpl

```diff
@@ -0,0 +1,21 @@
+use FileSystem;
+use MasonPublish;
+use MasonUtils;
+use Spawn;
+use TOML;
+
+
+proc main(){
+  runCommand('mason new testdir',true);
```

Comment:
Can we switch this to a mason library call rather than command line so that we don't require mason to be built to run it?